apps/ui/src/helpers/constants.ts

```diff
@@ -159,3 +159,9 @@ export const VALIDATION_TYPES_INFO: Record<
     description: 'Use EAS attest.sh to determine if user can create a proposal.'
   }
 };
+
+export const PROPOSALS_NETWORK_WHITELIST: ChainId[] = [
+  1, //Ethereum
+  42161, // Arbitrum
+  137 // Polygon
+];
```

Comment:
maybe we should add a param `premium` or `subscribed` on networks.json instead of a separate list?

Reply:
whitelist is the method proposed while discussing the task.

Having a better source of truth would be good, it should be the same source for UI and sequencer, and preferably, something dynamic, and not hardcoded, to avoid updating dependencies of snapshot.js on each update.

A better way would be to return a list of premium networks from the hub new `options` endpoint, or have new `premium` property on the `network` object on the hub.

@bonustrack wdyt ?

Reply:
Yes we can use the `networks` operation on the GraphQL API for it